docs/ContainerCustomization.md

```diff
@@ -33,6 +33,14 @@ By default, if your project has a RuntimeIdentifier set, that value will be used
 <ContainerRuntimeIdentifier>linux-x64</ContainerRuntimeIdentifier>
 ```
 
+> **Note**
+> If you'd like to publish to a musl-based OS like alpine (as opposed to a libc-based OS), you will need to specify the base image _exactly_, instead of relying on
```

Comment:
I'd say it's not exact (which to me would be e.g. `7.0.2-alpine3.17-amd64`).
```suggestion
> If you'd like to publish to a musl-based OS like alpine (as opposed to a libc-based OS), you will need to specify the base image _including architecture_, instead of relying on
```